cookbook/enums.md

```diff
@@ -0,0 +1,51 @@
+# Enums
+
+Enums belong to entityInfo part of metadata, which means using a particular schema version does not mean enum values won't change. Adding an enum value is considered a non breaking change and should be allowed. Removing or renaming an enum is a breaking change and should not be allowed.
+
+Considering the above, you have 2 options when designing model classes in a language which supports enum types (such as java):
+* don't use enum type, i.e. define enum metadata fields as strings in the model classes or
+* implement enum types which support unknowns.
+
+An example of a java enum with pojo2java nad java2pojo conversion support for the unknowns:
```

Comment:
You're right, it can incorporate only one unknown. Could incorporate any number if we resign from the pojo<->json conversion support. That means if enum value is unknown, you will not be able to save the document back. Or do you see a way to handle this?